### ISSUE TITLE
examples

### ISSUE THREAD

**Issue by muveso** [mnarimani/VContainerExtensions#2]
@pnarimani can you please share minimal examples?

**Comment by pnarimani** [comment]
Hi,
I no longer work on this repo since VContainer didn't work out for us in our company. We switched back to Zenject. 
However, If you want to try it out, here's how you use the classes:
`PrefabFactory` is used like `PlaceholderFactory` in Zenject. Create a factory class that inherits from the `PrefabFactory` and set up the parameters.  
Use `RegisterPrefabFactory` method to register it to the container. 
There's also pooling available but it's not perfect. The way you convert a factory to pooled is by providing `PoolSettings` in `RegisterPrefabFactory` method. There's also `IPoolable` interface available to receive callbacks for when an object goes in and out of the pool.

Cheers.

**Comment by muveso** [comment]
@pnarimani thanks for you return!

may I ask you why did you roll back to Zenject? VContainer is more flexible I think.

**Comment by pnarimani** [comment]
* Zenject is generally a more stable and bug free product (compare the issues page in github)
* There's no object validation graph in VContainer
* You cannot modify registrations after they were added in VContainer (I added this feature in my fork of VContainer.)
* Zenject has MonoInstaller which makes it easier to share binding code with different Containers/LifetimeScopes. (I added this feature in my fork of VContainer).
* We needed some of binding modes that existed in Zenject but VContainer didn't have them. (For example, NonLazy binding).
* There are some bugs with lifetime scope parenting. For example, If you bind multiple instances of interface `ITest` in parent scope, you couldn't resolve an array of `ITest` in child scope. I reported this bug but we changed to zenject so I didn't track to see if it's fixed or not.
* VContainer had a critical bug with `RegisterInstance` where it wouldn't work as expected in Android builds. I never tried to find the reason for this bug. We simply changed to Zenject.
